### DISCUSSION TITLE
The Usage about ClientPlayNetworking.registerReceiver​() in networking-v1

### DISCUSSION THREAD

**Discussion by Sekakou** [FabricMC/fabric-api#2454] [Mod Dev Support]
I used ClientPlayNetworking.registerReceiver​() to register a listener when the player entered a server,but the arguments I passed in were never executed.I've browsed through the sample mod on GitHub, but it doesn't work either.
Below is a screenshot of my code(api version:0.42.0+1.16(.5)
![image](https://user-images.githubusercontent.com/100136138/183336010-a1f5364d-c3e7-4704-b77b-83b3df6c50a4.png)

**Comment by Player3324** [comment]
I believe we fixed this in more modern versions (https://github.com/FabricMC/fabric/commit/442de8b8625915d8503533f039df686644b45ca5?), backports to old ones are limited to contributions (PRs) though.

**Reply by Sekakou** [reply]
hmm will this affect .registerReceiver​()?
